src/components/common/base-card/base-card.styles.tsx

```diff
@@ -13,7 +13,6 @@ interface IBodyStyleProps {
 // =============================================================================
 export const Body = styled.div<IBodyStyleProps>`
     padding: 1rem;
-    cursor: pointer;
```

Comment:
should keep this as by removing this style, all other buttons are showing the default arrow cursor.

I believe your component is not able to receive the styles when you convert it to a styled-component? in order to allow `BaseCard` to be a styled-component, it merely needs to receive the `className` prop

```
interface IProps {
  // ...
  className?: string;
}

export const BaseCard = ({ className, id, children, focused, onClick }: IProps) => {
    return (
        <Body
            tabIndex={0}
            $focused={focused}
            onClick={onClick}
            id={id}
            className={className}
            role="button"
        >
            {children}
        </Body>
    );
};

```